lib/find_retryables.ts

```diff
@@ -95,23 +107,43 @@ const main = async () => {
 
       const messages = await arbL2TxReceipt.getL1ToL2Messages(l3Provider)
 
+      if (messages.length == 0) {
+        console.log(`No retryable is found!`)
+        break
+      } else {
+        console.log(
+          `${messages.length} retryable${
+            messages.length === 1 ? '' : 's'
+          } found, checking their status. Arbtxhash: ${process.env
+            .ARBISCAN!}${l2TxHash}`
+        )
+        console.log('************************************************')
+      }
+
       for (let msgIndex = 0; msgIndex < messages.length; msgIndex++) {
         const message = messages[msgIndex]
+        const retryableTicketId = message.retryableCreationId
 
         let status = await message.status()
         // Logging different statuses of L2-to-L3 messages
         if (status === L1ToL2MessageStatus.NOT_YET_CREATED) {
-          console.log(`Ticket still not created: arbTxHash: ${l2TxHash}`)
+          console.log(`Ticket still not created:\narbTxHash: ${l2TxHash}`)
         } else if (status === L1ToL2MessageStatus.CREATION_FAILED) {
           console.log(
-            `☠️ Severe error: Retryable ticket creation failed: arbTxHash: ${l2TxHash}`
+            `☠️ Severe error: Retryable ticket creation failed:\norbitTxHash: ${l3Chain.explorerUrl}/tx/${retryableTicketId}`
           )
         } else if (status === L1ToL2MessageStatus.FUNDS_DEPOSITED_ON_L2) {
-          console.log(`⚠️ Ticket Not redeemed: arbTxHash: ${l2TxHash}`)
+          console.log(
+            `⚠️ Ticket Not redeemed:\norbitTxHash: ${l3Chain.explorerUrl}/tx/${retryableTicketId}`
+          )
         } else if (status === L1ToL2MessageStatus.EXPIRED) {
-          console.log(`Ticket expired (!): arbTxHash: ${l2TxHash}`)
+          console.log(
+            `Ticket expired (!):\norbitTxHash: ${l3Chain.explorerUrl}/tx/${retryableTicketId}`
+          )
         } else if (status === L1ToL2MessageStatus.REDEEMED) {
-          console.log(`Ticket is succesfully redeemed: arbTxHash: ${l2TxHash}`)
+          console.log(
+            `Ticket is succesfully redeemed:\norbitTxHash: ${l3Chain.explorerUrl}/tx/${retryableTicketId}`
+          )
         }
       }
```

Comment:
Nitpick: adding a console.log('') here will separate the info from the different l2TxReceipts